### PR TITLE
Added scripts to build and install the aws-sleek-transformer

### PR DESCRIPTION
**Pull request by iuliana**
*Description of changes:*

Added `.nvmrc` to fix version of `nodejs` used when building the project. The version in this file is used by `nvm` to install `nodejs` and `npm`. 

 `nvm` is used by the `install.sh` and the `Makefile` to install dependencies, build the project and install the resulting `aws-sleek-transformer` on user's system.

----

Tested on macOS and Ubuntu 24.04.

----
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+SHELL=bash
```
Comment (elamaran11) [inline Makefile:1]: Could we have a command to `npm publish` which i can use as owner of the NPM module?

File: README.md [hunk 2]
```diff
@@ -72,14 +77,80 @@ USAGE
...
+  TODO:
+  * Run validation before creating the issue
+
+
+EXAMPLES
+  $ aws-sleek-transformer create-issue filename
+```
+
+_See code: [src/commands/create-issue.ts](https://github.com/aws-samples/addons-transformer-for-amazon-eks/blob/v0.0.1/src/commands/create-issue.ts)_
+
```
Comment (elamaran11) [inline README.md:116]: Could we have an example option with `-f` `-d` option.

File: install.sh [hunk 3]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# check prerequisites NVM and AWS credentials
+if [ -d "${HOME}/.nvm/.git" ];
+then
+  echo "nvm installed";
+else
+  echo "nvm not installed. Install it as instructed here: https://github.com/nvm-sh/nvm#install--update-script";
```
Comment (elamaran11) [inline install.sh:8]: We should also run `aws sts get-caller-identity` to check if the terminal has AWS access.

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@iuliana Looks great. I have some strategic and tactical feedback.

File: Makefile (hunk 1 above)
Comment (iuliana) [reply Makefile:1]: sure :)

Comment (iuliana) [reply Makefile:1]: Added AWS access check.

File: install.sh (hunk 3 above)
Comment (iuliana) [reply install.sh:18]: Added AWS check.

**Review by elamaran11 — APPROVED** [review]
LGTM. Nice work @iuliana !

**Pull request merged**